md/dates-management-tutorial.md

```diff
@@ -0,0 +1,67 @@
+# How to work with dates in forms and BDM
```

Comment:
I will add a note (in case a string is used in the business) on how to convert it to a date for connectors expecting a java.util.Date